addons/hr_holidays/models/hr_leave_type.py

```diff
@@ -94,6 +93,15 @@ def _model_sorting_key(self, leave_type):
     support_document = fields.Boolean(string='Supporting Document')
     accruals_ids = fields.One2many('hr.leave.accrual.plan', 'time_off_type_id')
     accrual_count = fields.Float(compute="_compute_accrual_count", string="Accruals count")
+    # negative time off
+    allows_negative = fields.Boolean(string='Allow Negative Cap', default=False)
+    max_allowed_negative = fields.Integer(string="Amount in Negative")
```

Comment:
This could be one field, where not allowing negative could be ``0`` for ``max_allowed_negative``. I don't see the real value in having to check a box, which then allows you to set the value, while we could simply display the value directly with default value 0. But this is a functional thing :)

Reply:
Also I think the field names could be a bit more descriptive. I have a hard time coming up with better names myself though. Maybe 'Max negative amount'. And then maybe we add a tooltip to explain what it does?

Reply:
I agree, but I think that it's way more visual for the user to have something to check along with the value field hidden if unused

Reply:
I'll add a tooltip, indeed it might be helpful